test/Confluent.Kafka.UnitTests/Serialization/Double.cs

```diff
@@ -0,0 +1,76 @@
+// Copyright 2016-2017 Confluent Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// Refer to LICENSE for more information.
+
+using Confluent.Kafka.Serialization;
+using System;
+using System.Collections.Generic;
+using Xunit;
+
+namespace Confluent.Kafka.UnitTests.Serialization
+{
+    public class DoubleTests
+    {
+        [Theory]
+        [MemberData(nameof(TestData))]
```

Comment:
Any reason to use MemberData rather than InlineData like in ByteArrayTests? It's only usefull when one of the data is not constant or if used multiple times.